### PR TITLE
[NETBEANS-1074] Module Review web.client.rest

### PR DESCRIPTION
**Pull request by gilbertoca**
Checked the rat-report.txt file
Change the license header of Bundle.properties
Add the licenseinfo.xml 
Change the license header of some files

### CODE REVIEW

File: web.client.rest/nbproject/project.properties [hunk 1]
```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```
Comment (junichi11) [inline web.client.rest/nbproject/project.properties]: Please don't modify this file.

File: web.client.rest/src/org/netbeans/modules/web/client/rest/Bundle.properties [hunk 2]
```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```
Comment (junichi11) [inline web.client.rest/src/org/netbeans/modules/web/client/rest/Bundle.properties]: Please don't modify this file.

Comment (gilbertoca) [reply web.client.rest/src/org/netbeans/modules/web/client/rest/Bundle.properties]: Reverted the changes

**Comment by junichi11** [comment]
We should sqush and merge if the travis result is OK.

**Pull request merged**